testonly/matchers/atleast.go

```diff
@@ -21,15 +21,15 @@ import (
 	"github.com/golang/mock/gomock"
 )
 
+type atLeastMatcher struct {
+	num int
+}
```

Comment:
I actually prefer the old ordering, where the only exported thing in the file is at the top.